react.js

```diff
@@ -102,6 +102,7 @@ module.exports = {
     'react/no-is-mounted': 'error',
     'react/no-multi-comp': 'off',
     'react/no-namespace': 'error',
+    'react/no-object-type-as-default-prop': 'error',
```

Comment:
```suggestion
    'react/no-object-type-as-default-prop': 'off',
```

Reply:
Why are you disabling this rule?
It could cause bugs if we don't follow the guidelines of the rule.